sources/packages/api/src/database/entities/disbursement-values.model.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  RelationId,
+} from "typeorm";
+import { ColumnNames, TableNames } from "../constant";
+import { DisbursementSchedule } from "./disbursement-schedule.model";
+import { DisbursementValueType } from "./disbursement-value-type";
+import { RecordDataModel } from "./record.model";
+
+/**
+ * Disbursements values for each schedule on a Student Application.
+ * For each disbursement date there are a few values associated with it
+ * that represents the loans and grants to be disbursed.
+ */
+@Entity({ name: TableNames.DisbursementValue })
+export class DisbursementValue extends RecordDataModel {
+  @PrimaryGeneratedColumn()
+  id: number;
+  /**
+   * Generic type of the value (e.g. Canada Loan or BC Loan).
+   */
+  @Column({
+    name: "value_type",
+    type: "enum",
+    nullable: false,
+  })
+  valueType: DisbursementValueType;
+  /**
+   * Specific code for the loan/grant to be disbursed (e.g. CSLF or BCSL).
+   */
+  @Column({
+    name: "value_code",
+    nullable: false,
```

Comment:
just a general question. do we need to create an enum for code?

Reply:
The **_valueType_** enum created was created with the idea to allow us to identify what and how to map the grants and loans to be sent to ESDC.

The idea to leave the **_valueCode_** as a string is to allow the workflow to dynamically insert new grants/loans without any other code change on DB or API.